spp_programs_auto_enroll/wizard/create_program_wizard.py

```diff
@@ -0,0 +1,100 @@
+# Part of OpenG2P. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import _, fields, models
+
+# from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class SPPCreateNewProgramWiz(models.TransientModel):
+    _inherit = "g2p.program.create.wizard"
+
+    auto_enroll = fields.Boolean(default=False)
```

Comment:
```suggestion
    auto_enroll = fields.Boolean()
```
boolean field default is False